updates/2024-02-12-editorial-board-update.md

```diff
@@ -0,0 +1,119 @@
+# Meeting Notes: Python Docs Editorial Board
+
+## February 12, 2024
+
+## Attendees
+
+Mariatta, Ned, Joanna, Guido
+
+
+## Agenda
+
+1. What project do we want to deliver first? Here are some ideas to start the discussion:
+
+a. Define scope and outline for new intro tutorial
+b. Define scope and outline for new landing page
+c. Inventory/audit a section of the docs
+d. Personas (see #3 below)
+
+
+2. What does the docs community need from us most right now? Here are some ideas to start the discussion:
+
+- How-to discussion
+- First-person discussion 
+- A more thorough and more skimmable style guide
+
+3. [meta] Do we have a way to collect the largish topics they’ve discussed that we could provide guidance for? -> see editorial-board GH repo
+
+- Function parameters: bulleted or not
+- Examples: how to integrate into the docs
+- Type hints 
+
+4. Do we have an idea of what type of learner/user we need to optimize the docs for?
+
+- Wait until Carol is back 
+
+5. Do we have an idea of what type of contributor we want to prioritize removing obstacles for and creating systems for?
+
+6. Will the editorial board give an update at the language summit or PyCon?
+
+7. The language summit isn’t even announced yet. We’ll think about it next time
+
+## Notes
+
+1. Let’s begin with a relatively small project. 1b from above. It is concrete, and we can propose implementation and refine it quickly.
+
+2. 2a is a bigger issue, but is important. (will affect 2b)
+
+3. Recommendation: don’t use first-person for new docs.
+
+4. Howto doc: we might preserve some historical content. Some docs may be marked as “historical doc”, keep them, but might get changed. Add a disclaimer at the top.
+
+5. Add reminder to future contributors not to use those as a template.
+
+6. PR churn. Docs changes don’t need to follow the same guidelines.
+
+7. Style guide:
+   - In devguide now, update it
+   - https://devguide.python.org/documentation/style-guide/
+   - Some things should be in the style guide
+   - First person, terminologies like master/slave, should be there
+   - Big O notation should be in the Howto part of the style guide
+   - The EB will own this (and create issues and PRs)
+   - Should still be on devguide
+
+8. Discovery: [Docs Style Guide](https://docs.google.com/document/d/1rSkUIGFuI5zDMYturyJCo9agYPQ-d53Z3MntSV0rB3E/edit#heading=h.hflx14peuyef)
+
+9. Mariatta to create a Gdrive folder to be shared with PDEB
```

Comment:
```suggestion
9. Mariatta to create a Google Drive folder to be shared with PDEB
```
Any reason why a (private?) Gdrive folder is being used instead of something else (e.g. hackmd)?  Is it only to have a document accessible and editable by a restricted number of people?

Reply:
Yes.